src/styles/style.css

```diff
@@ -0,0 +1,99 @@
+@import url("https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,400;0,500;0,700;1,700&display=swap");
+ 
+
+.card_page {
+  font-family: "Roboto", sans-serif;
+}
+
+.container {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  background: #fff;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  width: 200px;
+  padding: 0 16px;
+}
+
+.container__img {
+  align-self: center;
+  margin-top: 32px;
+  min-height: 134px;
+  max-width: 160px;
+  margin-bottom: 40px;
+}
+
+.container__description {
+  margin-bottom: 4px;
+  color: #060b35;
+  line-height: 18px;
+  font-size: 12px;
+  font-weight: 500;
+}
+
+.container__productNumber {
+  margin-bottom: 16px;
+  color: #616070;
+  line-height: 14px;
+  font-size: 10px;
+  font-weight: 400;
+}
+
+.container__stars {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  margin-bottom: 24px;
+}
+
+.container__stars-review {
+  font-weight: 400;
+  font-size: 10px;
+  line-height: 14px;
+  color: #060b35;
+}
+
+.container__price {
+  display: flex;
+  justify-content: space-between;
+  margin-bottom: 16px;
+}
+
+.container__price-priceTag {
+  font-weight: 400;
+  line-height: 18px;
+  font-size: 12px;
+  color: #616070;
+}
+
+.container__price-price {
+  font-weight: 700;
+  font-size: 16px;
+  line-height: 18px;
+  color: #060b35;
+}
+
+.container__button {
+  color: #fff;
+  font-size: 14px;
+  font-weight: 700;
+  line-height: 16px;
+  text-transform: uppercase;
+  text-decoration: none;
+  text-align: center;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+
+  height: 40px;
```

Comment:
the checklist, point 8

Reply:
In today's lessons, Paul also used height: 40 px, how I can set proper height without hardcore it?

Reply:
line-height + paddings 🙂